05/Validator.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+class Validator {
+
+    public static function string($value, $min = 1, $max = INF) {
```

Comment:
Może warto dobrać lepszą nazwę metody. Tutaj zdecydowanie sprawdzamy długość ciagu znaków, a nie sam string. Lepiej może nazwać tak: `hasValidLength` lub rozszerzyć działanie tej metody i zostawić `string` np.  ` public static function string($value, $options) {` gdzie `$options` to różne opcje.